gennav/planners/rrt/rrt.py

```diff
@@ -60,15 +32,15 @@ def plan(self, start_point, goal_point, obstacle_list, animation=False):
         """Plans path from start to goal avoiding obstacles.
 
         Args:
-            start_point: tuple with start point coordinates.
-            end_point: tuple with end point coordinates.
+            start_point: Point with start point coordinates.
```

Comment:
Try to be specific here with `gennav.utils.geometry.Point` since even shapely has a point object. This applies to all docstrings

Reply:
Yeah, makes sense. On it.